src/tree/CosmosDBAccountProvider.ts

```diff
@@ -74,7 +74,17 @@ export class CosmosDBAccountProvider implements IChildProvider {
     private async initChild(client: CosmosDBManagementClient, databaseAccount: DatabaseAccount): Promise<IAzureTreeItem> {
         const defaultExperience = <API>databaseAccount.tags.defaultExperience;
         const resourceGroup: string = azureUtils.getResourceGroupFromId(databaseAccount.id);
-        const label: string = `${databaseAccount.name} (${getExperience(defaultExperience).shortName})`;
+        let accountKind;
+        try {
+            accountKind = getExperience(defaultExperience).shortName;
```

Comment:
I don't like this because getExperience could throw a RangeError in a bunch of different places (aka it's error-prone). Just change getExperience so that it doesn't throw a RangeError